Makefile.am

```diff
@@ -4,13 +4,24 @@
 
 ACLOCAL_AMFLAGS = -I m4
 
+if BUILD_OWN_PTHREAD_WORKQUEUES
```

Comment:
I had tried variations on this (and I tried this variation just now), but annoyingly it doesn't quite work.  The autoreconf and configure steps work correctly.  But in the generated top-level Makefile we end up with the SUBDIRS definition with the PTHREAD_WORKQUEUE_SUBDIR variable defined textually before PTHREAD_WORKQUEUE_SUBDIR.  This causes 'make' to not recurse into libpwq and the library doesn't get built.

I'm certainly not an autotools guru, but I wasn't able to get anything to work where SUBDIRS included a conditionally defined variable.